### ISSUE TITLE
Problem with boostrap with 5.0.0-rc1 version

### ISSUE THREAD

**Issue by nickamorg** [angular-fullstack/generator-angular-fullstack#2713]
<img width="510" alt="bootstrap_problem" src="https://user-images.githubusercontent.com/33521268/39092769-8a671934-461d-11e8-88c4-dd0fa31ceef6.png">
nd feature requests.

generator-angular-fullstack | 5.0.0-rc1
Node | 8.11.1
npm | 5.6.0
Operating System | Windows 10

Transpiler | Babel
Markup | HTML
CSS | CSS
Router | There wasn't any option
Client Tests | Mocha
DB | MongoDB
Auth | Y
Bootstrap | Y

Despite the fact that I have included the Bootstrap option(and UI Bootstrap), there is no result on the page, either the code when checking with inspector.

Is there any recommedation about that?

Best regards

**Comment by nickamorg** [comment]
> Do you get any errors?

No, only some warnings on the client side, like so: http://prntscr.com/j8up3o

> Is the bootstrap CSS being injected onto the page?

No, it isn't in the page.
Also, I can't find how it is imported or how it should be imported.
With the previous versions I could find it in the file "client/app/app.js", but not anymore.

**Comment by Awk34** [comment]
If you selected bootstrap, it should be imported directly in the CSS file: https://github.com/angular-fullstack/generator-angular-fullstack/blob/master/templates/app/client/app/app(css).css#L2
